packages/esm-appointments-app/translations/en.json

```diff
@@ -84,7 +84,7 @@
   "errorCreatingAppointmentService": "Error creating appointment service",
   "expected": "Expected",
   "gender": "Gender",
-  "highestServiceVolume": "High volume service: {{time}}",
+  "highestServiceVolume": "Highest volume service: {{time}}",
```

Comment:
Side note... when one updates a translation list this (but not the key) I assume, if works as it did before, Transifex invalidates all the translations for this key.  (This is not always what we want, but what I'd say we want in this case).